py/src/braintrust/test_framework.py

```diff
@@ -250,28 +251,33 @@ def exact_match(input, output, expected):
     def simple_scorer(input, output, expected):
         return {"name": "simple_scorer", "score": 0.8}
 
-    result = await Eval(
-        "test-no-logs",
-        data=[{"input": "hello", "expected": "hello world"}, {"input": "test", "expected": "test world"}],
-        task=lambda input_val: input_val + " world",
-        scores=[exact_match, simple_scorer],
-        no_send_logs=True,
-    )
-
-    # Verify it returns results
-    assert len(result.results) == 2
-    assert result.results[0].input == "hello"
-    assert result.results[0].output == "hello world"
-    assert result.results[0].scores["exact_match"] == 1.0
-    assert result.results[0].scores["simple_scorer"] == 0.8
-
-    assert result.results[1].input == "test"
-    assert result.results[1].output == "test world"
-    assert result.results[1].scores["exact_match"] == 1.0
-    assert result.results[1].scores["simple_scorer"] == 0.8
-
-    # Verify it builds a local summary (no experiment_url means local run)
-    assert result.summary.project_name == "test-no-logs"
-    assert result.summary.experiment_url is None
-    assert result.summary.scores["exact_match"].score == 1.0
-    assert result.summary.scores["simple_scorer"].score == 0.8
+    # Spy on the logger's init function to verify no experiment is created
+    with patch("braintrust.framework.init_experiment") as mock_init:
+        result = await Eval(
```

Comment:
for python we can use something like:

https://github.com/braintrustdata/braintrust-sdk/blob/445cba50d36e46a64605ecbca35a76c94a528db3/py/src/braintrust/test_logger.py#L981-L984